app/javascript/src/editable/editable_utilities.js

```diff
@@ -0,0 +1,121 @@
+
```

Comment:
Should this file exist?

Most of these methods could exist within the `EditableContent` class?  But some of them are used within `content.js` too.

Or, these functions could live in the global `utilities.js` file?